.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "freyr-js-dev",
+    "image": "mcr.microsoft.com/devcontainers/javascript-node",
+    "features": {
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {}
+    },
+    "postCreateCommand": {
+        "yarn install": "yarn global add freyr",
+        "install build tools": "sudo apt-get update && sudo apt install cmake -y"
```

Comment:
Never dealt with devcontainers before, but does it presuppose a debian-based environment? what if apt-get is unavailable?

Reply:
Yea it is a debian container with some basic dependencies installed. The image details are here:
https://hub.docker.com/_/microsoft-devcontainers-javascript-node?tab=description